lotus-soup/deals_e2e.go

```diff
@@ -108,3 +127,50 @@ func dealsE2E(t *testkit.TestEnvironment) error {
 	t.SyncClient.MustSignalAndWait(ctx, testkit.StateDone, t.TestInstanceCount)
 	return nil
 }
+
+// filToAttoFil converts a fractional filecoin value into AttoFIL, rounding if necessary
+func filToAttoFil(f float64) big.Int {
+	a := mbig.NewFloat(f)
+	a.Mul(a, mbig.NewFloat(float64(build.FilecoinPrecision)))
+	i, _ := a.Int(nil)
+	return big.Int{Int: i}
+}
+
+func initPaymentChannel(t *testkit.TestEnvironment, ctx context.Context, cl *testkit.LotusClient, minerAddr testkit.MinerAddressesMsg) error {
```

Comment:
Can we move this to the `testkit` library under a `paych.go` file, and make it public?
Then we can dedup this logic in the `paych_stress` and be DRYer.